source/macros.tex

```diff
@@ -317,13 +318,13 @@
 
 %% Miscellaneous
 \newcommand{\uniquens}{\placeholdernc{unique}}
-\newcommand{\stage}[1]{\item{\textbf{Stage #1:}}\xspace}
-\newcommand{\doccite}[1]{\textit{#1}\xspace}
+\newcommand{\stage}[1]{\item{\textbf{Stage #1:}}}
```

Comment:
Separate issue: Should use square brackets: `\item[Stage #1:]` and adjust formatting of items in `description` environments. (Note that the `description` environment is also used in `preprocessor.tex` in a weird way, too.)

Reply:
Done, thanks!